modules/installation-aws-user-infra-rhcos-ami.adoc

```diff
@@ -6,13 +6,11 @@
 [id="installation-aws-user-infra-rhcos-ami_{context}"]
 = {op-system} AMIs for the AWS infrastructure
 
-Red Hat provides {op-system-first} AMIs valid for the various Amazon Web
-Services (AWS) zones you can specify for your {product-title} nodes.
+Red Hat provides {op-system-first} AMIs that are valid for the various AWS zones that you can manually specify for your {product-title} nodes.
```

Comment:
Debated whether to include a note before or after this table that using stream metadata is the recommended method for easier and more reliable automation processes?

Reply:
nit: `s/zones/regions/`